gui/src/components/Balances.tsx

```diff
@@ -16,20 +16,24 @@ export function Balances() {
     "db_get_erc20_balances",
     { address }
   );
+  const { data: settings } = useInvoke<GeneralSettings>("settings_get");
+
+  const reorderedBalances = (balances || [])
+    .map<[`0x${string}`, bigint]>(([c, b]) => [c, BigInt(b)]) // is it possible to get the proper type from backend directly?
+    .filter(([, balance]) => (settings?.hideEmptyTokens ? !!balance : true))
+    .sort(([, a], [, b]) => (b > a ? 1 : b < a ? -1 : 0));
```

Comment:
What do you think of using the `sortBy` from lodash? I feel like it's more readable since you can just return the balance and don't need to write these conditions

Reply:
I actually think we shouldn't have this `sort` right now
firstly because sorting by balance is very arbitrary, and will mean the list changes order with every transaction

secondly because we probably want to have some other order defined by the backend, e.g. by the order in which they were first detected (which is what metamask does)

Reply:
yeah, i get that. my idea was just to bring the tokens with balance (!== 0) to the top in case you're not filtering those